tools/permutate_axis/permutate_axis.xml

```diff
@@ -0,0 +1,30 @@
+<tool id="permutate_axis" name="Permutate Axis" version="0.1">
+  <description> Permutates axes <description/>
+  <requirements>
+    <requirement type="package" version="0.20.0" >scikit-image</requirement>
+    <requirement type="package" version="1.15.4">numpy</requirement>
+  <requirements/>
+  <command>
+    <![CDATA[
+    python "$__tool_directory__/permutate_axis.py" '$input' '$output' --axis $axis $permutate
+    ]]>
+  </command>
+  <inputs>
+    <param name="input" type="data" format="tiff,png,jpg,bmp" label="Binary Image File"/>
+    <param name="permutate" type="text" default="[0,1,2]" label="new channel order"/>
```

Comment:
can we get rid of the brackets here and add a validator to only allow numbers and comma?

Reply:
@znerp feel free to change the tool to get rid of the brackets.